src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -56,22 +57,35 @@ private void DeleteFakeCsProj(FileInfo fakeCsproj)
 			fakeCsproj.Delete();
 		}
 
-		private void FixXunit(DirectoryInfo directory)
+		private void FixTests(DirectoryInfo directory)
 		{
 			WriteLine("Fixing SpecFlow generated files for xUnit 2");
 
 			var glueFiles = directory.GetFiles("*.feature.cs", SearchOption.AllDirectories);
 
 			foreach (var glueFile in glueFiles)
 			{
-				WriteLine("Fixed: " + glueFile.FullName);
+				WriteLine("Fixing: " + glueFile.FullName);
 				var content = File.ReadAllText(glueFile.FullName);
-				content = content.Replace(" : Xunit.IUseFixture<", " : Xunit.IClassFixture<");
-				content = content.Replace("[Xunit.Extensions", "[Xunit");
+				content = FixXUnit(content);
+				content = FixMsTest(content);
 				File.WriteAllText(glueFile.FullName, content);
 			}
 		}
 
+		private static string FixMsTest(string content)
+		{
+			content = Regex.Replace(content, ".*Microsoft.VisualStudio.TestTools.UnitTesting.Description.*", "");

```

Comment:
I think it's the cause of the ugly mstest names, but i dont know how to fix it or if a fix exist as I found no similar attribute.